tests/utils/dir-is-empty.test.js

```diff
@@ -0,0 +1,9 @@
+test('dir is empty', () => {
```

Comment:
The name of test case should show what it's testing - in this case we're testing - 'dir is not empty'